### DISCUSSION TITLE
Atuin

### DISCUSSION THREAD

**Discussion by indibart** [catppuccin/catppuccin#2513] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://docs.atuin.sh/guide/theming/

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by ghost** [comment]
Wow that actually looks really fun. Expression of interest.

However I won't be able to start work for a few weeks.

**Comment by uncenter** [comment]
Atuin should just be using your term colors by default anyway, so if you use a Catppuccin terminal theme it is already themed.

**Comment by 0mykull** [comment]
will make a port of this.

**Comment by sgoudham** [comment]
Closing as further discussion should take place in #2757
